src/views/Home/components/DiningBalance/index.jsx

```diff
@@ -138,9 +148,20 @@ const DiningBalance = () => {
           spacing={0}
           style={{ paddingTop: 5, paddingBottom: 10 }}
         >
-          <Grid item>
+          <Grid item sx={10}>
             <Typography variant="body2" className={styles.label2}>
               {diningInfo.ChoiceDescription}
+              <IconButton
+                variant="body2"
+                title="Change Meal Plan"
+                className={styles.label2}
+                //className={styles.gc360_my_profile_edit_icon}
```

Comment:
```suggestion
                className={styles.label2}
                //className={styles.gc360_my_profile_edit_icon}
```
I think we might need a better name than label2, also delete the other style tag if not needed

Reply:
This naming was given by the style team and this style file we are getting from is being used in different components in the Home folder. I can still change it if you want to

Reply:
removed the other style tag